docs/packaging/prepare-for-packaging.md

```diff
@@ -84,25 +84,27 @@ mkdir ~/solus-builds
 cd solus-builds
 ```
 
-##  Recommended aliases
-Consider creating these aliases in your `.bashrc` file (or equivalent for the zsh or fish shell).
+## Fork the getsolus/packages Repository
 
-The script `common/Scripts/yauto.py` is used to create an initial `package.yml` file for packages that aren't yet in the repository.
+Create your own fork of [getsolus/packages](https://github.com/getsolus/packages) using the GitHub web UI or [`gh` cli tool](https://cli.github.com/manual/gh_repo_fork) from the `github-cli` package. It will be forked to `github.com/yourgithubaccount/packages`.
 
-```bash
-alias fetchYml="$HOME/solus-builds/common/Scripts/yauto.py"
-```
+## Clone Your Forked Package Repository
 
-The script `common/Scripts/epcsearch.py` is used to find packages that satisfy build dependencies.
+Create a local clone of the package repository you just forked
 
 ```bash
-alias epcsearch='~/packaging/common/Scripts/epcsearch.py'
+cd ~/solus-builds
+gh repo clone yourgithubaccount/packages
 ```
 
-The `yupdate` tool is used when updating packages.
+## Recommended aliases
+
+Consider creating these aliases in your `.bashrc` file (or equivalent for the zsh or fish shell).
+
+The script `common/Scripts/epcsearch.py` is used to find packages that satisfy build dependencies.
```

Comment:
Maybe while we're at it we could also link this to the more detailed explanation in https://github.com/getsolus/help-center-docs/blob/master/docs/packaging/packaging-practices.md#finding-what-package-provides-a-dependency-if-any ?

Reply:
Actually I just realized we already have a go-task command for this

```
pkgconfig:             Find which package provides a given pkgconfig target.

Example usage: `go-task pkgconfig -- Qt5Core Qt6Core`
```

Reply:
Didn't realized the `pkgconfig` command was the new equivalent there. Deleted the rest of "recommended aliases".

We have a link to the "packaging practices" page on the "creating a new package" page after the initial package.yml output